README.md

```diff
@@ -1,65 +1,283 @@
+      --volumes 16                            \
+      --capacity 16Ti
 
 ```
-kubectl minio tenant expand --name tenant1 --servers 8 --volumes 32 --capacity 32Ti
+
+- The `minio-tenant-1` argument specifies the name of the existing MinIO Tenant to expand.
+
+- The `--servers` field indicates the number of `minio` pods to deploy into the cluster.
+  The cluster *must* have at least one available Node per `minio` pod. 
+
+- The `--volumes` field indicates the total number of volumes in the Tenant. MinIO
+  generates a Persistent Volume Claim (`PVC`) for each volume and evenly distributes
+  volumes across each `minio` pod. The example above results in 4 volumes per `minio` pod.
+
+- The `--capacity` field indicates the total capacity of the cluster. MinIO determines the
+  amount of storage to request for each `pvc` by dividing the specified capacity by the
+  total number of volumes in the server. The example above results in 1Ti requested
+  capacity per volume.
+
+
+# Kubernetes Cluster Configuration
+
+## Default Storage Class
+
+The MinIO Kubernetes Plugin (`kubectl minio`) automatically generates
+Persistent Volume Claims (`PVC`) as part of deploying a MinIO Tenant. 
+The plugin defaults to creating each `PVC` with the `default`
+Kubernetes [`Storage Class`](https://kubernetes.io/docs/concepts/storage/storage-classes/).
+
+MinIO Tenants *require* that the `StorageClass` set 
+`volumeBindingMode` to `WaitForFirstConsumer`. The default `StorageClass` may use the 
+`Immediate` setting, which can cause complications during `PVC` binding. MinIO 
+strongly recommends creating a custom `StorageClass` for use by 
+`PV` supporting a MinIO Tenant:
+
+
+The following `StorageClass` object contains the appropriate fields for use with the MinIO Plugin:
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+    name: local-storage
+provisioner: kubernetes.io/no-provisioner
+volumeBindingMode: WaitForFirstConsumer
 ```
 
-This will add 32 drives spread uniformly over 8 servers to the tenant `tenant1`, with additional capacity of 32Ti. Read more about [tenant expansion here](https://github.com/minio/operator/blob/master/docs/expansion.md).
+To specify the storage class, include the `--storageClassName` option to 
+`kubectl minio tenant create`.
+
+
+## Local Persistent Volumes
+
+MinIO automatically creates Persistent Volume Claims (PVC) as part of Tenant creation.
+Ensure the cluster at least one 
+[Persistent Volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)
+for each PVC MinIO requests.
+
+You can estimate the number of PVC by multiplying the number of nodes in the 
+Tenant by the number of drives per node. For example, a 4-node Tenant with
+4 drives per node requires 16 PVC and therefore 16 PV.
+
+MinIO *strongly recommends* using locally-attached storage for each PV for
+object storage performance. MinIO recommends the following CSI drivers for
+creating local PV:
+
```

Comment:
```suggestion
MinIO *strongly recommends* recommends the following CSI drivers for creating local PV:
```

Reply:
Resolved in separate commit to crunch two lines together.